components/charts/line-charts/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { ResponsiveLine } from '@nivo/line'
+
+type LineChartProps = {
+    data: any
```

Comment:
No 😄 we know the `data` should always have the same types to expect.

An array of objects that has an 
1. `id`
2. `color`
3. data object with `x`, `y`

Also, let's rename the `data` prop to be `dataSource` instead